### ISSUE TITLE
This extension requires the Microsoft ODBC Driver 11 or 13 for SQL Server.

### ISSUE THREAD

**Issue by bkraul** [Microsoft/msphpsql#695]
## Driver version or file name
mssqlodbc version 17.0.0001.1 Linux
sqlsrv extension 4.3.0 (latest)

## SQL Server version
SQL Server Standard 2012 (11.0.6607.3).

## Client operating system
Windows 10 x64

## PHP version
7.0.25-0ubuntu0.16.04.1

## Microsoft ODBC Driver version
17.0.0001.1 Linux

## Problem description
Even though latest drivers and extensions are installed, I keep getting the error:
```
Error information:
SQLSTATE: IMSSP
Code: -49
Message: This extension requires the Microsoft ODBC Driver 11 or 13 for SQL Server. Access the following URL to download the ODBC Driver 11 or 13 for SQL Server for x64: http://go.microsoft.com/fwlink/?LinkId=163712
SQLSTATE: IM002
Code: 0
Message: [unixODBC][Driver Manager]Data source name not found, and no default driver specified
```

## Expected behavior and actual behavior
Connection should be established to the desired database. Instead, it errors with the message described above.

## Repro code
```php
<?php

ini_set('display_errors', 1);
ini_set('display_startup_errors', 1);
error_reporting(E_ALL);

$host = "integrator.epginsurance.com";
$connectionOptions = array(
  "Database" => "mydb",
  "UID" => "myuser",
  "PWD" => "mypassword"
);


// establish the connection.
$conn = sqlsrv_connect( $host, $connectionOptions);
if( $conn === false ) {
    die( FormatErrors( sqlsrv_errors()));
}

//Select Query
$tsql= "SELECT @@Version as SQL_VERSION";
//Executes the query
$getResults= sqlsrv_query( $conn, $tsql );
//Error handling
 
if ( $getResults == FALSE )
    die( FormatErrors( sqlsrv_errors()));
?> 
 <h1> Results : </h1>
 <?php
while ( $row = sqlsrv_fetch_array( $getResults, SQLSRV_FETCH_ASSOC )) {
    echo ( $row['SQL_VERSION']);
    echo ("<br/>");
}
sqlsrv_free_stmt( $getResults );
function FormatErrors( $errors )  
{  
    /* Display errors. */  
    echo "Error information: <br/>";  
  
    foreach ( $errors as $error )  
    {  
        echo "SQLSTATE: ".$error['SQLSTATE']."<br/>";  
        echo "Code: ".$error['code']."<br/>";  
        echo "Message: ".$error['message']."<br/>";  
    }  
}  
```

**Comment by yitam** [comment]
Yes @bkraul we have been getting issues like this lately due to the upgrade of ODBC driver from 13.1 to 17.
You can either upgrade the PHP drivers to our latest RC or downgrade ODBC to 13.1

See related issue #684 for details.

**Comment by bkraul** [comment]
Is this going to continue to be an issue as you upgrade drivers? I know that when this started there was no support for connecting to SQL Server 2005, which made us move to our 2012 server. I would hate have to be made to move to 2017 because that is the minimum version the driver supports. It is actually kind of surprising.

**Comment by bkraul** [comment]
Closed as a duplicate of #684

**Comment by yitam** [comment]
@bkraul we are aware of this problem and ODBC drivers team is actively addressing this issue. 
On the other hand, this has nothing to do with SQL Server version, and as of today we still support SQL Server 2012.
